tests/unit/hooks/test_zabbix.py

```diff
@@ -0,0 +1,194 @@
+from enum import Enum
+
+from flexmock import flexmock
+
+import borgmatic.hooks.monitor
+from borgmatic.hooks import zabbix as module
+
+server = 'https://zabbix.com/zabbix/api_jsonrpc.php'
+itemid = 55105
+username = 'testuser'
+password = 'fakepassword'
+api_key = 'fakekey'
+host = 'borg-server'
+key = 'borg.status'
+value = 'fail'
+
+data_host_key = {
+    "jsonrpc": "2.0",
+    "method": "history.push",
+    "params": {"host": host, "key": key, "value": value},
+    "id": 1,
+}
+
+data_itemid = {
+    "jsonrpc": "2.0",
+    "method": "history.push",
+    "params": {"itemid": itemid, "value": value},
+    "id": 1,
+}
+
+data_user_login = {
+    "jsonrpc": "2.0",
+    "method": "user.login",
+    "params": {"username": username, "password": password},
+    "id": 1,
+}
+
+auth_headers_api_key = {
+    'Content-Type': 'application/json-rpc',
+    'Authorization': f'Bearer {api_key}'
+}
+
+auth_headers_username_password = {
+    'Content-Type': 'application/json-rpc'
+}
+
+def test_ping_monitor_config_with_api_key_only():
```

Comment:
Part of the convention in borgmatic test function names is to briefly describe _what_ you're asserting or testing for, not just the input that goes into the particular test. So like: `test_ping_monitor_config_with_api_key_only_bails()` or `...exits_early()` or even `...does_not_call_server()`.